packages/ai/src/ai/modules/task/task_engine.py

```
@@ -1661,6 +1695,12 @@ async def start_task(self) -> None:
 
         except Exception as e:
             await self._terminated()
+            tracker.on_failed(self.id)
+            _logger.error(
```

Ruff (0.15.12)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)